lalrpop/src/lr1/codegen/base.rs

```diff
@@ -222,16 +221,13 @@ impl<'codegen, 'grammar, W: Write, C> CodeGenerator<'codegen, 'grammar, W, C> {
                     self.prefix, self.prefix, user_type_parameters,
                 ),
                 format!(
-                    "{}TOKENS: IntoIterator<Item={}TOKEN>",
-                    self.prefix, self.prefix
+                    "{}TOKENS: IntoIterator<Item={}TOKEN> {}",
+                    self.prefix,
+                    self.prefix,
+                    if self.repeatable { "+ Clone" } else { "" }
```

Comment:
Nit: `"+ Clone"` -> `" + Clone"` and then you can remove the space at the end of `"{}TOKENS: IntoIterator<Item={}TOKEN> {}"`